### DISCUSSION TITLE
Do I have false expectations or did I mess up?

### DISCUSSION THREAD

**Discussion by KatieBelli** [voicepaw/so-vits-svc-fork#645] [Q&A]
I trained for about **4264 epochs now** (Britney Spears model) and my settings are that it saves the model after 80-100 epochs, depending on my current settings/file amount. 
I have audio files from other singers and use the latest saved model on them to compare the progress.

I noticed that the voice sounds similiar for many epochs now but **the higher frequencies still sound a bit robotic** and this has only improved a bit over the last 2000 epochs.

**Is it normal that the voice model doesn't sound natural in general or am I just too impatient?**

[I searched on Youtube for some AI covers by Britney Spears and most of them sound so good and clean, like Britney sung those songs (like Genie in a Bottle cover by AIVerse channel).]

_My current config file (lines which I changed but could be that some values are not changed)_:

>     "log_interval": 100,
>     "eval_interval": 3000,
>     "seed": 10965,
>     "epochs": 7001,
>     "learning_rate": 0.00015,
> 
>     "batch_size": 5,
>     "num_workers": 5,

_My GPU_: 
NVIDIA GeForce GTX 1660

_Current amount of files (<10 seconds each file)_: 
172 singing files (24 minutes 34 seconds) + 20 speaking files (2minutes 10seconds) = 192 files (26 minutes 44 seconds)

Currently, all files don't have any reverb, delay, clicks or noise and their quality is lossless. Breathing is included in most of them.

I added some new audio files a bit before epoch 808, 883 and 3474. 
I found on reddit how to do this:

> -save them to your dataset_raw folder (optionally save a backup of your config file too)
> -run following commands: svc pre-resample, svc pre-config svc pre-hubert
> -edit new config file in configs folder to reflect previous settings/learning rate
> -you should see python files in the dataset folders for your speakers for each .wav now
> -you can now safely run svc train -t

Before epoch 1236 I cleaned some exisiting audio files of remaining hall that already existed before epoch 808 (that hall was not that noticeable but I want the cleanest dry vocals) and some noise. The quality of the vocals in each file remains very high quality without distortion after cleaning them.

I hope that this is enough information for you to tell me if I messed up or if my expectations are too high.

Any suggestions of changing values etc. are welcome.

**Comment by KatieBelli** [comment]
Here are some errors that appear when I run the train command, maybe they are the cause of this issue?

![01](https://github.com/voicepaw/so-vits-svc-fork/assets/109644887/23f4a650-4777-4ddc-985f-f98ef56d94cf)

![02](https://github.com/voicepaw/so-vits-svc-fork/assets/109644887/683f8bc6-a499-4b17-a887-e623d7f9a474)

**Comment by hataori-p** [comment]
I have a similar experience when training on data which comes from a stem separation SW. (I assume you don't have direct recordings of Britney). It seems to me that the model picks up some distortion from the separation SW what we don't hear and mess up the high frequency noises (in my case). Now I don't consider the singing data separated from songs HQ.

This is not an answer, only my experience from one model training.

**Reply by KatieBelli** [reply]
> I have a similar experience when training on data which comes from a stem separation SW. (I assume you don't have direct recordings of Britney). It seems to me that the model picks up some distortion from the separation SW what we don't hear and mess up the high frequency noises (in my case). Now I don't consider the singing data separated from songs HQ.
> 
> This is not an answer, only my experience from one model training.

I have recordings of her: official stems (dry vocals: only her solo voice, no backing vocals or layered vocals) and files of her recording songs. When I filtered out remaining hall I made sure that the filtered hall didn't influence the quality of the vocals.

**Reply by hataori-p** [reply]
I'm sorry for assuming things. And how is it that there was a hall in it?

**Reply by KatieBelli** [reply]
> I'm sorry for assuming things. And how is it that there was a hall in it?

It's alright, I don't think it's problematic to assume.

I also was surprised since they record in a studio in a room which is isolated so there can't be any hall or delay when they record their vocals.

The hall is very little, quieter and short, so it's not that noticeable if you don't pay close attention while listening. It's more obvious if the singer sings a bit louder and less noticeable if the singer sings quieter.

**Comment by sbersier** [comment]
You can try to increase the "noise scale" in svcg (let's say --> 0.9). It might help tuning down (a bit) the high-frequencies "robotic" artefacts.
